classes/publication/PKPPublication.inc.php

```diff
@@ -94,8 +99,9 @@ public function getFullTitles() {
 	 * @return string
 	 */
 	public function getLocalizedTitle($preferredLocale = null) {
-		$prefix = $this->getLocalizedData('prefix', $preferredLocale);
-		$title = $this->getLocalizedData('title', $preferredLocale);
+		$usedLocale = null;
+		$title = $this->getLocalizedData('title', $preferredLocale, $usedLocale);
+		$prefix = $this->getData('prefix', $usedLocale);
```

Comment:
I find the use of references in this way to be counterintuitive and worry about it getting misunderstood and refactored away down the line. I don't really see an alternate approach, though...

Reply:
It's not the most intuitive idiom, but there are widely-used precedents (e.g. `exec`, `getimagesize`, etc) with very similar philosophies, i.e. do-a-thing-but-optionally-allow-richer-feedback (as we're doing here).